site-src/index.md

```diff
@@ -20,54 +20,119 @@ variety of networking use-cases.
 
 ![Gateway API Model](./images/api-model.png)
 
-
-How do Gateway API improve upon current standards like Ingress?
-
-- **More expressive** - They express more core functionality for things like
-header-based matching, traffic weighting, and other capabilities that were only
-possible in Ingress through custom means.
-- **More extensible** - They allow for custom resources to be linked at various
-layers of the API. This allows for more granular customization at the
-appropriate places within the API structure.
-- **Role oriented** - They are separated into different API resources that map
-to the common roles for running applications on Kubernetes.
-- **Generic** - This isn't an improvement but rather something
+Following are some of the design goals where Gateway aims to improve upon 
+current standards like Ingress.
+
+- **Role-oriented** - Gateway is composed of API resources which model 
+organizational roles that use and configure Kubernetes service networking. 
+A role-oriented API allows diverse teams to operate on shared network 
+infrastructure. Gateway is designed to balance the flexibility and control of 
+these roles in a way that reflects the culture and structure of the organization
+ using it.
+- **Portable** - This isn't an improvement but rather something
 that should stay the same. Just as Ingress is a universal specification with
 [numerous implementations](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/),
-Gateway API are designed to be a portable specification supported by many
+Gateway API is designed to be a portable specification supported by many
 implementations.
+- **Expressive** - Gateway API resources support core functionality for things 
+like header-based matching, traffic weighting, and other capabilities that 
+were only possible in Ingress through custom annotations.
+- **Extensible** - Gateway API allows for custom resources to be linked at 
+various layers of the API. This makes granular customization possible at the
+appropriate places within the API structure.
 
-Some other notable capabilities include …
-
-- **Shared Gateways** - They allow the sharing of load balancers and VIPs by
-permitting independent Route resources to bind to the same Gateway. This allows
-teams to share infrastructure safely without requiring direct coordination.
-- **Typed backend references** - With typed backend references Routes can
-reference Kubernetes Services, but also any kind of Kubernetes resource that is
-designed to be a Gateway backend.
-- **Cross-Namespace references** - Routes across different Namespaces can bind
-to Gateways. This allows for shared networking infrastructure despite Namespace
-segmentation for workloads.
-- **Classes** - GatewayClasses formalize types of load balancing implementations.
-These classes make it easy and explicit for users to understand what kind of
-capabilities are available as a resource model itself.
+### Why does a role-oriented API matter?
```

Comment:
Both this section and the one below are great and well written, but they might be a bit too much content for the main page. Maybe a new "Why Gateway?" page or similar?

Reply:
I slimed it down to about 50% of what it was before, lmk if that's small enough